lib/child_process.js

```diff
@@ -537,8 +537,8 @@ function normalizeSpawnArguments(file, args, options) {
 }
 
 
-var spawn = exports.spawn = function spawn(/* file, args, options */) {
-  var opts = normalizeSpawnArguments.apply(null, arguments);
+var spawn = exports.spawn = function spawn(file, args, options_) {
+  var opts = normalizeSpawnArguments(file, args, options_);
   var options = opts.options;
```

Comment:
A little confused, is it suitable to keep the parameter as `options` instead of `options_`(maybe for consistency with the documentation), and just use `options = opts.options;` to reassign `options`, avoid re-var `options` ?

Reply:
I think this is a valid point as the argument names should match the ones in the documentation.

Reply:
I'll update the PR later, but to be clear, the current function signature doesn't match the docs 😄

Reply:
The comment does 😄.